src/main/resources/migration/changelog-v5.3.0.xml

```diff
@@ -0,0 +1,2298 @@
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="PROJECT_ACCESS_TEAMS">
+            <column name="PROJECT_ID" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="TEAM_ID" type="BIGINT"/>
+        </createTable>
+
+        <createTable tableName="MANAGEDUSERS_TEAMS">
+            <column name="TEAM_ID" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="MANAGEDUSER_ID" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="POLICY_PROJECTS">
+            <column name="POLICY_ID" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="PROJECT_ID" type="BIGINT"/>
+        </createTable>
+
+        <createTable tableName="POLICY_TAGS">
+            <column name="POLICY_ID" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="TAG_ID" type="BIGINT"/>
+        </createTable>
+
+        <createTable tableName="SCHEMAVERSION">
+            <column autoIncrement="true" name="ID" startWith="2" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="SCHEMAVERSION_PK"/>
+            </column>
+            <column name="VERSION" type="VARCHAR(255)"/>
+        </createTable>
+
+        <createTable tableName="INSTALLEDUPGRADES">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="INSTALLEDUPGRADES_PK"/>
+            </column>
+            <column name="ENDTIME" type="TIMESTAMP WITH TIME ZONE"/>
+            <column name="STARTTIME" type="TIMESTAMP WITH TIME ZONE"/>
+            <column name="UPGRADECLASS" type="VARCHAR(255)"/>
+        </createTable>
+
+        <createTable tableName="CONFIGPROPERTY">
+            <column autoIncrement="true" name="ID" startWith="71" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="CONFIGPROPERTY_PK"/>
+            </column>
+            <column name="DESCRIPTION" type="VARCHAR(255)"/>
+            <column name="GROUPNAME" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="PROPERTYNAME" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="PROPERTYTYPE" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="PROPERTYVALUE" type="VARCHAR(1024)"/>
+        </createTable>
+
+        <createIndex indexName="WORKFLOW_STATE_N49" tableName="WORKFLOW_STATE">
+            <column name="PARENT_STEP_ID"/>
+        </createIndex>
```

Comment:
There are still some indexes here with `_N[0-9]+` notation. We should fix those names as well.

Reply:
hawk eye :D  .. and i couldn't spot it even after creating a spreadsheet for it. Fixing now